meerk40t/gui/laserrender.py

```diff
@@ -782,13 +782,15 @@ def measure_text(self, node):
         @param node:
         @return:
         """
-
-        bmp = wx.Bitmap(1000, 500, 32)
+        dimension_x = 1000
```

Comment:
We could probably get away with a smaller initial box if we're actually guessing from the revision factor to start with. The box is so big because we're trying to avoid the bounds.

Reply:
Agree, that was a 45 seconds fix